src/lib.rs

```diff
@@ -91,78 +91,24 @@
 #![doc(html_logo_url = "https://raw.githubusercontent.com/nvzqz/cargo-emit/assets/logo.svg?sanitize=true")]
 
 #![deny(missing_docs)]
-#![no_std]
 
-mod rerun;
-mod rustc;
+mod pair;
+mod rerun_if_changed;
+mod rerun_if_env_changed;
+mod rustc_cdylib_link_arg;
+mod rustc_cfg;
+mod rustc_env;
+mod rustc_flags;
+mod rustc_link_lib;
+mod rustc_link_search;
+mod warning;
```

Comment:
Thanks for moving macros into their own files. Much easier to navigate and read test outputs.

Reply:
Glad you're okay with it. I figured the `rustc.rs` file was already plenty long and I absolutely wanted the tests to be alongside the macros, rather than in a `/tests/smoke.rs` file or something. So giving them their own file, each felt like it would solve that issue and also make discoverability much easier.